### PR TITLE
sci-biology/MuSeqBox: add 5.7.2021, drop 3.4.2020 

### PR DESCRIPTION
**Pull request by lucasmitrak**
* Add new version 5.7.2021
* Source code for old version 3.4.2020 is no longer available

### CODE REVIEW

**Comment by lucasmitrak** [comment]
Let me know if I should have done two branches/PRs for this. I can remake it

File: sci-biology/MuSeqBox/MuSeqBox-5.7.2021.ebuild [hunk 1]
```diff
@@ -1,20 +1,19 @@
...
 HOMEPAGE="http://brendelgroup.org/bioinformatics2go/MuSeqBox.php"
 SRC_URI="http://www.brendelgroup.org/bioinformatics2go/Download/MuSeqBox-${PV//./-}.tar.gz"
-# code will move to https://github.com/BrendelGroup
 
 LICENSE="GPL-3"
 SLOT="0"
 KEYWORDS="~amd64"
 
-S="${WORKDIR}/MUSEQBOX5.5"
+S="${WORKDIR}/MUSEQBOX5.7"
```
Comment (AndrewAmmerlaan) [inline sci-biology/MuSeqBox/MuSeqBox-5.7.2021.ebuild:16]: Here we can use on the version manipulation functions to make future version bumps a bit easier, e.g.:
```
S="${WORKDIR}/MUSEQBOX$(ver_cut 1-2)"
```
See here for the full overview of all available version manipulation functions: https://mgorny.pl/articles/the-ultimate-guide-to-eapi-7.html#version-manipulation-and-comparison-commands

**Comment by AndrewAmmerlaan** [comment]
> Let me know if I should have done two branches/PRs for this. I can remake it

It's fine like this :+1:

**Comment by lucasmitrak** [comment]
> 
> 
> > Let me know if I should have done two branches/PRs for this. I can remake it
> 
> It's fine like this 👍

After I make this commit, should I squash them? The first commit is adding version 5.7.2021, and then I dropped the old version 3.4.2020, so I cannot just squash the last two commits.

**Comment by lucasmitrak** [comment]
More specifically, how should I precede from here in terms of git? I am reading that tutorial and I do not know which option is applicable, if any.

**Comment by AndrewAmmerlaan** [comment]
> > > Let me know if I should have done two branches/PRs for this. I can remake it
> > 
> > 
> > It's fine like this +1
> 
> After I make this commit, should I squash them? The first commit is adding version 5.7.2021, and then I dropped the old version 3.4.2020, so I cannot just squash the last two commits.

Yes please squash the version bump commit and the follow-up commit together. You can actually squash/fixup commits which are not directly following one another. E.g.:
```
git rebase -i HEAD~3
```
Will show you the last three commits, e.g.:
```
pick introduce-new-version
pick drop-old-version
pick small-fix-to-new-version
```
By changing both the order of the commits (third becomes the second), and changing pick to squash you can squash the first and third (now second) commit together.
```
pick introduce-new-version
squash small-fix-to-new-version
pick drop-old-version
```
Note that instead of squash you can use fixup to discard the commit message.
Next you can force push to this branch, to overwrite the commits already in this PR with the updated ones.

**Comment by lucasmitrak** [comment]
> 
> 
> > > > Let me know if I should have done two branches/PRs for this. I can remake it
> > > 
> > > 
> > > It's fine like this +1
> > 
> > 
> > After I make this commit, should I squash them? The first commit is adding version 5.7.2021, and then I dropped the old version 3.4.2020, so I cannot just squash the last two commits.
> 
> Yes please squash the version bump commit and the follow-up commit together. You can actually squash/fixup commits which are not directly following one another. E.g.:
> 
> ```
> git rebase -i HEAD~3
> ```
> 
> Will show you the last three commits, e.g.:
> 
> ```
> pick introduce-new-version
> pick drop-old-version
> pick small-fix-to-new-version
> ```
> 
> By changing both the order of the commits (third becomes the second), and changing pick to squash you can squash the first and third (now second) commit together.
> 
> ```
> pick introduce-new-version
> squash small-fix-to-new-version
> pick drop-old-version
> ```
> 
> Note that instead of squash you can use fixup to discard the commit message.
> Next you can force push to this branch, to overwrite the commits already in this PR with the updated ones.

Ok, I did it. That was way easier than I thought it would be.

**Comment by AndrewAmmerlaan** [comment]
> Ok, I did it. That was way easier than I thought it would be.

Excellent, Thank You :+1:

**Pull request closed without merging**